modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/tasks/RepairRequestTask.java

```diff
@@ -288,14 +290,18 @@ public RepairJob(Map<VersionedKey, Map<UUID, VersionedValue>> data, String cache
                                 cacheObjCtx,
                                 ownersNodesSize);
 
-                            keyWasSuccessfullyFixed = ignite.cache(cacheName).withKeepBinary().<RepairEntryProcessor.RepairStatus>invoke(
+                            EntryProcessorResult<Object> invoke = ctx.cache().keepBinary().invoke(
```

Comment:
Why not to use ignite.cachex(cacheName) instead of ctx.cache()?
I'd also consider adding withKeepBinary to IgniteInternalCache in order to ease testing, issue related to mocking, cause keepBinary() itself in final.

Reply:
Done, cachex#keepBinary is not final, I've managed to mock it, I think there is no need in adding #withKeepBinary to internalCache anymore.